### PR TITLE
[openvino] 2023.3.0 release

### PR DESCRIPTION
**Pull request by RaguzovaTatyana**

### CODE REVIEW

**Comment by RaguzovaTatyana** [comment]
@microsoft-github-policy-service agree company="Intel"

**Comment by Cheney-W** [comment]
```
CMake Error at scripts/cmake/z_vcpkg_apply_patches.cmake:34 (message):
  Applying patch failed: Checking patch
  cmake/developer_package/version.cmake...
  Hunk #2 succeeded at 19 (offset 3 lines).
  error: while searching for:
  endfunction()
  function(ov_commit_number VAR REPO_ROOT)
      if(GIT_FOUND)
          execute_process(
                  COMMAND ${GIT_EXECUTABLE} rev-list --count --first-parent HEAD
                  WORKING_DIRECTORY ${REPO_ROOT}
  error: patch failed: cmake/developer_package/version.cmake:27
  error: cmake/developer_package/version.cmake: patch does not apply
```

File: ports/openvino/portfile.cmake [hunk 1]
```diff
@@ -1,8 +1,8 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO openvinotoolkit/openvino
-    REF "${VERSION}"
-    SHA512 d44aa06763b18fabb6e9fb52834cbe9b6c91384541177093741519c7cf6f209720cb9aabf0b4a2b2b32625399ca2076d7b1a3430ea00dadeda86255fb8baf720
+    REF "ceeafaf64f346c6f14a67c612e131da5c27ef620"
+    SHA512 0e572ce17fb5e7fbfa845d7d7c4a9d480fd8f674f6900e0243a365f099e5d018c7a68ce26dfdab7c9ad4c62e9285d8ae0493712bb1db5e2f5aa6d18143f9c99c
     PATCHES
         # vcpkg specific patch, because OV creates a file in source tree, which is prohibited
         001-disable-tools.patch
```
Comment (ilya-lavrenov) [inline ports/openvino/portfile.cmake:8]: @RaguzovaTatyana let's drop `002-git-version.patch` because it's already part of the product

**Review by ilya-lavrenov — APPROVED** [review]

**Comment by Cheney-W** [comment]
@RaguzovaTatyana Is your pull request ready to be merged? If yes, please click the `Ready for review` button.

**Comment by RaguzovaTatyana** [comment]
@Cheney-W, Yes, It's already ready to be merged. I was just waiting for the release day.

**Pull request merged**